charts/tensorleap/templates/rabbitmq-cm.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-config
+data:
+  enabled_plugins: |
+    [rabbitmq_management].
+  rabbitmq.conf: |
+    log.file.level = debug
+    log.console = true
+    log.console.level = debug
+    queue_master_locator = min-masters
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-login
+data:
+  erl_cookie: cookie123
```

Comment:
1. Move to the other config map
2. name it `erlang_cookie`, I'm not sure that everybody will understand `erl`

Reply:
Are you sure that you want this? The other config map is mounted to the pod, this way a file named `erlang_cookie` will be created and not used..

Reply:
IIUC a file will not be created because you choose what to take
```
volumes
- ...
- name: rabbitmq-config
  configMap:
    name: rabbitmq-config
    items:
      - key: enabled_plugins
        path: enabled_plugins
      - key: rabbitmq.conf
        path: rabbitmq.conf
```

Reply:
So I should duplicate it w/ a new name and only the one key?

Reply:
I think
1. Add `erlang_cookie: cookie123` to `rabbitmq-config`
2. Delete `rabbitmq-login`
3. In the _StatefulSet_ change `rabbitmq-login` to `rabbitmq-config` and `erl_cookie` to `erlange_cookie`

Reply:
Done